cypress/e2e/pages/stainingQC.cy.ts

```diff
@@ -131,5 +131,42 @@ describe('Staining QC', () => {
         });
       });
     });
+
+    describe('when Pretreatment QC is selected', () => {
+      before(() => {
+        cy.visit('/lab/staining_qc');
+      });
+
+      it('should  not display measurement fields', () => {
+        selectOption('qcType', 'Pretreatment QC');
+        selectSGPNumber('SGP1008');
+        cy.get('#labwareScanInput').type('STAN-411{enter}');
+        cy.findAllByTestId('coverage').should('have.length', 0);
+      });
+      it('should not display Pass/Fail icons', () => {
+        cy.findAllByTestId('passIcon').should('have.length', 0);
+      });
+      it('has comment dropdowns enabled', () => {
+        getAllSelect('comment').forEach((elem: any) => {
+          cy.wrap(elem).should('be.enabled');
+        });
+      });
+      context('when changing the comment all dropdown', () => {
```

Comment:
will it better if  we move the page setting /loading checks  to unit tests ?

Reply:
Good point. But,  I have only added unit tests for LabwareResult, and  these tests need to be covered as part of StainingQC  unit tests which doesn't exist. I will see whether it can be added